### ISSUE TITLE
Can no longer create profiles when running on testnet

### ISSUE THREAD

**Issue by gfodor** [bitclout/frontend#356]
As of two weeks ago, pulling down core, backend, and frontend allowed setting up testnet just fine, with a mined key then being able to set a profile, and post.

Now, when trying to save the profile, I get a RuleErrorTxnMustHaveAtLeastOneInput error. Looking at the POST and tracing, the transaction has no cost so there is no Input/Output created. The workaround I found is to patch frontend to have a default transaction fee per KB of non-zero.

**Comment by maebeam** [comment]
Are you using the `--regtest` param?

**Comment by maebeam** [comment]
Are you using the `--regtest` param?

**Comment by gfodor** [comment]
yes the underlying testnet node is running with `--regtest` 

note: i did notice it gossiping with some seed peers I didn't recognize unless i specified `private-mode` -- here's the full command line

```
builds/<my custom bitclout node> run \
  --glog-v=0 \
  --glog-vmodule="*api*=0,*balance*=0,*frontend*=0,*peer*=0,*addr*=0,*network*=0,*utils*=0,*connection*=0,*main*=0,server*=0,*mempool*=0,*miner*=0,*blockchain*=0,*block_producer*=1" \
  --add-ips=localhost:19000 \
  --testnet=true  \
  --regtest \
  --api-port=18001 \
  --protocol-port=18000 \
  --txindex \
  --block-producer-seed='essence camp ghost remove document vault ladder swim pupil index apart ring' \
  --starter-bitclout-seed='road congress client market couple bid risk escape artwork rookie artwork food' \
  --access-control-allow-origins=http://localhost:4200,http://localhost:80,http://localhost:18002 \
  --secure-header-allow-hosts=localhost:4200 \
  --secure-header-development=true \
  --block-cypher-api-key=092dae962ea44b02809a4c74408b42a1 \
  --min-satoshis-for-profile=0 \
  --show-processing-spinners=true \
  --private-mode=true
```

**Comment by gfodor** [comment]
Also I discovered this morning this error also occurs when posting.

**Comment by maebeam** [comment]
For some reason I'm unable to reproduce this. Does this happen with an empty / fresh data directory?

**Comment by andrewarrow** [comment]
ran into same thing, here is delta that made frontend work again:

```
+++ b/src/app/global-vars.service.ts
@@ -100,7 +100,7 @@ export class GlobalVarsService {
 
   // map[pubkey]->bool of globomods
   globoMods: any;
-  feeRateBitCloutPerKB = 0.0;
+  feeRateBitCloutPerKB = 0.01;
```

**Comment by maebeam** [comment]
Reproduction steps would be great, I tried a fresh n0_test and didn't experience this

**Comment by maebeam** [comment]
This will be fixed by #378
